### PR TITLE
Updated footer to new design

### PR DESCRIPTION
**Pull request by kelvinkipruto**
This PR Redesigns the footer as requested.
> NB: Similar to PesaYetu
footer2
footer2

footer3

### CODE REVIEW

**Comment by kilemensi** [comment]
Does the design affect mobile/small screens as well @kelvinkipruto? If yes, please add screenshots for those as well.

**Comment by kelvinkipruto** [comment]
> Does the redesign affect mobile/small screens as well @kelvinkipruto? If yes, please add screenshots for those as well.

Updated

File: src/app/footer/footer.component.scss [hunk 1]
```diff
@@ -98,9 +97,26 @@
...
     }
+
+    .social-links {
+      margin-top: 20px;
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      a {
+        margin-right: 10px;
+      }
```
Comment (kilemensi) [inline src/app/footer/footer.component.scss:108]: Last `a` shouldn't have any margin-right.

**Review by kilemensi — COMMENTED** [review]
👍🏽 

----

Haven't seen the updated designs but going by the implementation here:

1. Mobile: **Resources** and **Stay in touch** (heading at least) needs to be centered.
2. Desktop: Summary needs to be left-aligned.
3. Icons looks quite think/light. Since we're limiting them to 30px, can we use the 32px ones instead of 64px ones?
Screenshot from 2022-11-29 11-08-11

File: src/app/footer/footer.component.scss [hunk 2]
```diff
@@ -98,9 +97,24 @@
       font: inherit;
       text-decoration: none;
     }
+
+    .social-links {
+      margin-top: 20px;
+      display: flex;
+      flex-direction: row;
```
Comment (kilemensi) [inline src/app/footer/footer.component.scss:104]:
```suggestion
      flex-direction: row;
      gap: 10px;
```

Spacing between icons.

**Review by kilemensi — COMMENTED** [review]
👍🏽 

---

On mobile, Resources needs to be centered (left image vs current)
S

On desktop, I think summary, resources and Stay in touch needs to be aligned (similar to PesaYetu)?
Screen Shot 2022-11-30 at 11 05 37

**Comment by kilemensi** [comment]
I’ve given my review @kelvinkipruto

**Pull request merged**